ro_help/hub/templates/base.html

```diff
@@ -9,7 +9,7 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.8.0/css/bulma.min.css">
     <link rel="stylesheet" href="{% static "css/bulma-collapsible.min.css" %}">
     <link rel="stylesheet" href="{% static "css/hub.css" %}">
-    {% include "favicons.html" %}
```

Comment:
you should not remove this line :)

Reply:
Deleted by mistake while fixing conflicts (: